frontend/src/features/properties/list/columns.tsx

```diff
@@ -4,9 +4,13 @@ import { ReactComponent as LandSvg } from 'assets/images/icon-lot.svg';
 import React from 'react';
 import { CellProps } from 'react-table';
 import { Link } from 'react-router-dom';
-import { formatMoney, formatNumber } from 'utils';
+import { formatMoney, formatNumber, mapLookupCode } from 'utils';
 import { IProperty } from '.';
 import { ColumnWithProps } from 'components/Table';
+import { ParentGroupedFilter } from 'components/SearchBar/ParentGroupedFilter';
```

Comment:
Why not use ParentSelect instead?

Reply:
The parent group filter seems to work better for me, and the ParentSelector breaks the code